tests/infra/remote_client.py

```diff
@@ -97,9 +97,11 @@ def stop(self, retrieve_csvs=False):
                         self.remote.get(csv, 1)
         except Exception:
             LOG.exception("Failed to shut down {} cleanly".format(self.name))
+            raise
```

Comment:
I don't think this is right, this will result in stop() not being executed for other remotes if a stop() raises.

Reply:
Good point!